File: Codigo_Fonte_pascal/lista3/lista3_6.pas
program busca_sequencial;

var
   lista : array[1..5] of integer;
   i, k, condiccao : integer;

begin
     condiccao := 0;
     for i := 1 to 5 do
          readln(lista[i]);

     write('Digite a chave: ');
     readln(k);

     for i:= 1 to 5 do
     begin
         if lista[i] = k then
            begin
            writeln('Chave encontrada da posicao: ', i);
            condiccao := 1;
            end;
     end;

     if condiccao = 0 then
        writeln('Chave ',k,' nao encontrada');
     readln;
end.

File: Codigo_Fonte_c/Lista 3/lista3_6.c
#include <stdio.h>

int main(){
	int lista[5];
	int i, k, condiccao = 0;
	printf("Digite a chave: ");
	scanf("%d", &k);
	for (i = 0; i < 5; i++){
		scanf("%d", &lista[i]);
	}
	for (i = 0; i < 5; i++){
		if (lista[i] == k){
			condiccao = 1;
			printf("Chave encontrada no endereço %d.\n", i);		
		}
	}
	if (condiccao ==  0){
		printf("Chave %d não encontrada.\n", k);	
	} 
	return 0;	
}
